### ISSUE TITLE
FAHClient cannot detect OpenCL correctly

### ISSUE THREAD

**Issue by fcole90** [fcole90/fah-snap#2]
In the FAHControl, checking System Info, for OpenCL it says
```
Not detected: Failed to open dynamic library 'libOpenCL.so': libOpenCL.so: cannot open shared object file: No such file or directory
```
However if I look for OpenCL vendors I can find the them
```
snap run --shell folding-at-home-fcole90.FAHClient
fabio@fabio-X550JK:/home/fabio/scratch/folding-at-home$ cat /etc/OpenCL/vendors/nvidia.icd 
libnvidia-opencl.so.1
```

It is possibly missing some symlink.

**Comment by widget** [comment]
````
07:42:58:******************************* System ********************************
07:42:58:        CPU: AMD Ryzen 5 3600X 6-Core Processor
07:42:58:     CPU ID: AuthenticAMD Family 23 Model 113 Stepping 0
07:42:58:       CPUs: 12
07:42:58:     Memory: 15.65GiB
07:42:58:Free Memory: 13.72GiB
07:42:58:    Threads: POSIX_THREADS
07:42:58: OS Version: 5.4
07:42:58:Has Battery: false
07:42:58: On Battery: false
07:42:58: UTC Offset: 1
07:42:58:        PID: 2409
07:42:58:        CWD: /var/snap/folding-at-home-fcole90/32
07:42:58:         OS: Linux 5.4.0-18-generic x86_64
07:42:58:    OS Arch: AMD64
07:42:58:       GPUs: 0
07:42:58:       CUDA: Not detected: Failed to open dynamic library 'libcuda.so':
07:42:58:             libcuda.so: cannot open shared object file: No such file or
07:42:58:             directory
07:42:58:     OpenCL: Not detected: clGetPlatformIDs() returned -1001
````

I also have this related error of not finding `libcuda.so`.

**Comment by fcole90** [comment]
Hi, I think this is a related, but a different bug. Can you open a new issue? In it, can you also tell me your OS version and the version of your CUDA libraries?
Note however, that I did not begin working on CUDA support yet.

**Comment by fcole90** [comment]
This should now work if you have the right interface connected `snap connect folding-at-home-fcole90:hardware-observe`. Note that if you didn't connect the interface before launching the application for the first time, it may be necessary that you delete the configuration files before starting it again. Please, let me know if there's any other issue on this.
